src/change_stream.ts

```diff
@@ -358,6 +405,91 @@ export interface ChangeStreamInvalidateDocument extends ChangeStreamDocumentComm
   operationType: 'invalidate';
 }
 
+/**
+ * Only present when the `showExpandedEvents` flag is enabled.
+ * @public
+ * @see https://www.mongodb.com/docs/manual/reference/change-events/#invalidate-event
```

Comment:
The see URLs need updating

Reply:
from slack: the docs aren't ready for us to consume yet.  I created a ticket for us to update these urls to point to the correct event documentation and linked it to the docsp ticket for these changes (https://jira.mongodb.org/browse/NODE-4295).